README.md

```diff
@@ -116,10 +116,14 @@ Rscript -e 'library(caret); install.packages(unique(modelLookup()[modelLookup()$
 
 #### Setting Up Local Env For Testing
 
-1. create Py 3.7 or 3.8 venv
-1. `pip install -r requirements_dev.txt`
-1. `pip install -e custom_model_runner/`
-1. pytest to your heart's content
+1. Create Py venv, >=3.9 is recommended.
```

Comment:
```suggestion
1. Create virtualenv  Python>=3.9
```
is this true?  i think we have conflicts somewhere with higher versions.

Reply:
done